### PR TITLE
GIX-1546: Integrate geo location service

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

We want to restrict participation of users from certain countries to participate in SNS swap.

This PR: Integrate with two location services that return the user location.

# Changes

* Implement two services in location api function.
* Add CSP rules for both new services.

# Tests

* Add test for location api function.

### CODE REVIEW

File: frontend/src/lib/components/project-detail/ProjectStatusSection.svelte [hunk 1]
```diff
@@ -68,7 +68,7 @@
     </div>
 
     <div class="actions content-cell-details">
-      {#if myCommitmentIcp !== undefined}
+      {#if nonNullish(myCommitmentIcp) && myCommitmentIcp.toE8s() > BigInt(0)}
```
Comment (dskloetd) [inline frontend/src/lib/components/project-detail/ProjectStatusSection.svelte:71]: Did this get mixed in from another PR?

File: frontend/src/tests/lib/components/project-detail/ProjectStatusSection.spec.ts [hunk 2]
```diff
@@ -42,6 +42,31 @@ describe("ProjectStatusSection", () => {
     expect(queryByTestId("sns-project-current-commitment")).toBeInTheDocument();
   });
 
+  it("should render user commitment", () => {
```
Comment (dskloetd) [inline frontend/src/tests/lib/components/project-detail/ProjectStatusSection.spec.ts:45]: This doesn't belong in this PR, right?

File: frontend/src/lib/components/project-detail/ProjectStatusSection.svelte (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/lib/components/project-detail/ProjectStatusSection.svelte:71]: No, it doesn't. I must have started this branch from the other PR.

Let me close this and open a new one. Thanks!

**Comment by lmuntaner** [comment]
Branch not from `main`
